README.md

```diff
@@ -0,0 +1,9 @@
+**Group Name: A-Cube**
```

Comment:
@sky-2002  Add repo info like what this repo is for etc, above this line

Reply:
Sure